src/request.ts

```diff
@@ -1039,6 +1039,14 @@ class DatastoreRequest {
           }
         }
 
+        if (entityObject.excludeLargeProperties) {
```

Comment:
I believe we need to document this option somewhere.

Reply:
done

Reply:
👍 Thanks! Is there a JSDocs / TypeScript place that we can put it, too?

Reply:
@stephenplusplus Hi i updated the JSDocs! hopefully you like it.